player.gd

```diff
@@ -92,7 +113,6 @@ func clamp_to_ladder():
 
 func _rollback_tick(delta, _tick, _is_fresh):
 	_force_update_is_on_floor()
-	$State.text = MovementState.keys()[movement_state]
```

Comment:
Thought we might want to have the movement state label kept to further debug issues